packages/elements/src/slider/elements/slider.ts

```diff
@@ -455,6 +455,10 @@ export class Slider extends ControlElement {
 
       marker.style.setProperty('left', `${markerPosition}%`);
 
+      if (!this.isShowInputField && !marker.labelAlign) {
+        marker.labelAlign = markerPosition === 0 ? 'left' : markerPosition === 100 ? 'right' : null;
```

Comment:
let's refactor this to `switch` instead for better readability.

Reply:
For me, I think that with just two cases, the ternary operator is more concise and straightforward.

Reply:
OK, this one is rather simple.  Marking as resolved. 👌🏼